src/autocomplete/PeopleAutocomplete.js

```diff
@@ -22,11 +22,26 @@ type Props = {
   userGroups: UserGroup[],
 };
 
+type HandleUserItemAutocompleteParams = {
+  fullName: string,
+};
+
 class PeopleAutocomplete extends PureComponent<Props> {
   props: Props;
 
+  handleUserGroupItemAutocomplete = (name: string): void => {
+    const { onAutocomplete } = this.props;
+    onAutocomplete(`*${name}*`);
+  };
+
+  handleUserItemAutocomplete = (params: HandleUserItemAutocompleteParams): void => {
+    const { onAutocomplete } = this.props;
+    const { fullName } = params;
+    onAutocomplete(`**${fullName}**`);
```

Comment:
This gets a bit verbose. I think this method body would be better as
```
    this.props.onAutocomplete(`**${params.fullName}**`);
```
Destructuring to give names to the various pieces is a helpful thing to do when the code is longer and when things get repeated; but here we use `this.props` just once and `params` just once, so the whole thing just gets longer by adding these intermediate names. And without them it all fits very plainly on one line, which is great.

(Better yet, combine with the remark I made on a different file, for
```
  handleUserItemAutocomplete({ fullName }) {
    this.props.onAutocomplete(`**${fullName}**`);
  }
```
.)